src/backend/joanie/core/utils/contract.py

```diff
@@ -149,3 +151,21 @@ def generate_zip_archive(pdf_bytes_list: list, user_uuid: str, zip_uuid=None) ->
     storage.save(name=zip_archive_name, content=zip_buffer)
 
     return zip_archive_name
+
+
+def organization_has_owner(order_uuid=None) -> bool:
```

Comment:
Sorry I misread the first time. WDYT of
```suggestion
def order_has_organization_owner(order_uuid=None) -> bool:
```

Reply:
I'm open to any suggestion as long as it is more readable. I feel like this is more accurate. Thank you for your suggestion, I will take it.